CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 **Bug Fixes**
 - Fixed a bug in parsing output of disk utilization summary
 - Fixed a bug in parsing SAS token not having '?' as first character
+- Fixed a bug in append file flow resolving data corruption
+- Fixed a bug in MSI auth to send correct resource string
```

Comment:
Chris Jiang's private msi endpoint test revealed that its not going correct. Instead of resource string we are putting endpoint url in mis auth request